client/src/frontend/components/composed/EntityCollection/patterns/JournalIssues.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import PropTypes from "prop-types";
+import memoize from "lodash/memoize";
+import lh from "helpers/linkHandler";
+import EntityThumbnail from "global/components/atomic/EntityThumbnail";
+import ThumbnailGrid from "global/components/composed/ThumbnailGrid";
+import { FooterLink, ProjectCollectionIcon } from "../parts";
+import EntityCollection from "../EntityCollection";
+import { getHeroImage, getHeaderLayout } from "../helpers";
+import { fixtures } from "helpers/storybook/exports";
+
+const mapIssues = memoize(journal => {
+  // TODO: update once API is in place
+  return fixtures.collectionFactory("issue", 4).map(issue => issue.data);
+  // return journal.relationships.collectionProjects.map(
+  //   cp => cp.relationships.project
+  // );
+});
+
+function getIssues(journal, limit) {
+  const issues = mapIssues(journal);
+  return issues.slice(0, limit);
+}
+
+function JournalIssuesEntityCollection({
+  journal,
+  paginationProps,
+  filterProps,
+  limit,
+  ...passThroughProps
+}) {
+  const { title, slug, descriptionFormatted: description } = journal.attributes;
+  const issues = getIssues(journal, limit);
+  const image = getHeroImage(journal);
+  const headerLayout = getHeaderLayout(journal);
+
+  // TODO: update once API is in place
+  // const totalIssueCount = journal.relationships.collectionProjects.length;
+  const totalIssueCount = issues.length;
+
+  return (
+    <EntityCollection
+      title={title}
+      description={description}
+      IconComponent={props => (
+        <ProjectCollectionIcon {...props} collection={journal} />
+      )}
+      image={image}
+      headerLayout={headerLayout}
+      headerLink={lh.link("frontendJournal", journal.id)}
+      BodyComponent={props =>
+        !!issues?.length && (
+          <ThumbnailGrid {...props}>
+            {({ stack }) =>
+              issues.map(item => (
+                <EntityThumbnail key={item.id} entity={item} stack={stack} />
+              ))
+            }
+          </ThumbnailGrid>
+        )
+      }
+      FooterComponent={() =>
+        totalIssueCount > limit && (
+          <FooterLink
+            to={lh.link("frontendJournals", slug)}
+            label="See all issues"
+          />
+        )
+      }
```

Comment:
In design, this sometimes has a "Visit the journal page" link rather than "See all issues". Maybe we could render the visit link if `totalIssueCount` is less than `limit`?

<img width="266" alt="Screen Shot 2022-01-18 at 11 10 39 AM" src="https://user-images.githubusercontent.com/32903719/150003289-f18d855e-4152-49a0-b56d-608ee7002ea4.png">

Reply:
Good eye! Yeah, that must be the logic for when those different labels appear.